torch/csrc/jit/tensorexpr/ir_printer.h

```diff
@@ -75,7 +75,11 @@ class TORCH_API IRPrinter : public IRVisitor {
   class PrinterStream : public std::ostream {
```

Comment:
Should we do this as a RAII where we restore the prior locale after we're done printing ?

Reply:
I think this should only affect one instance of the ostream; it doesn't affect global state.

... however, NVRTC _does_ set the global locale to C